packages/components/src/Heading/Heading.tsx

```diff
@@ -10,11 +10,15 @@ interface HeadingProps {
    */
   readonly level: HeadingLevel;
   readonly children: ReactNode;
+  /**
+   * @default "p"
```

Comment:
I think this default is incorrect. Looking at the code it would actually be `h<level>`

Reply:
Or maybe not specifying a default here. It could be something like
```
Allows overriding of the element rendered. If one isn't specified it will be a heading element matching the specified level
```

Reply:
🤦‍♂️  I forgot to delete that when I grabbed it from Typography. Yes, let me resolve this.

Reply:
We have all been there 😂

Reply:
Okay! Should be good now.